blog/2024-01-09-deprecate-invalid-sensor-uom-state-class-use.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Maybe rename the file with the date of today so matches when we'll post the blog better.